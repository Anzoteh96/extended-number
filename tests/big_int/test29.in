1044325074932528643298727821905451635948726592823036448974067744159972945948116211084047
634414102165559480015293452843501307702044643249532490128694638081417458017694303194791
