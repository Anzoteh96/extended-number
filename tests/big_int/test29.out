Enter two numbers n and m:
n is 1044325074932528643298727821905451635948726592823036448974067744159972945948116211084047
m is 634414102165559480015293452843501307702044643249532490128694638081417458017694303194791
n + m is 1678739177098088123314021274748952943650771236072568939102762382241390403965810514278838
n - m is 409910972766969163283434369061950328246681949573503958845373106078555487930421907889256
n * m is 662534554782300786200402542432972487797579369152504434329999873668334570379229390616005088275646025069281131495533162472126738140057859102090709765114411724838590957513599177
n / m is 1
n % m is 409910972766969163283434369061950328246681949573503958845373106078555487930421907889256
